### ISSUE TITLE
Broken links in UDP howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4274]
|<img alt="Screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Screwtape reported|
|-|-|
|Trac ID|trac#4274|
|Type|enhancement|
|Created|2010-02-12 09:44:23Z|
|Branch|https://github.com/twisted/twisted/tree/broken-udp-links-4274|

"frostschutz" just dropped by #twisted to mention that in the [UDP How To](http://twistedmatrix.com/documents/current/core/howto/udp.html), there are links to `twisted.internet.protocol.DatagramProtocol.startProtocol` and `twisted.internet.protocol.DatagramProtocol.stopProtocol` that actually lead to 404 errors.

It turns out, those methods are actually defined on `AbstractDatagramProtocol` so I'm not sure what the links should actually look like.

<details><summary>Searchable metadata</summary>

```
trac-id__4274 4274
type__enhancement enhancement
reporter__Screwtape Screwtape
priority__normal normal
milestone__ 
branch__branches_broken_udp_links_4274 branches/broken-udp-links-4274
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__easy documentation, easy
time__1265967863000000 1265967863000000
changetime__1267717639000000 1267717639000000
version__None None
owner__jesstess jesstess
cc__jesstess cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_2"></a>|@jesstess commented|
|-|-|

(In [28539]) Branching to 'broken-udp-links-4274'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

(In [28540]) Fix startProtocol and stopProtocol links and remove trailing whitespace.

refs [#4274](https://github.com/twisted/twisted/issues/4274)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @jesstess|
|-|-|

There are some more code references to class names in the document that could use a `<code>`.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_6"></a>|@jesstess commented|
|-|-|

(In [28615]) Add more code markup, spellcheck, be a little more precise
terminology-wise in some places, etc.

refs [#4274](https://github.com/twisted/twisted/issues/4274)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="TimAllen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|TimAllen set owner to TimAllen|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="TimAllen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_9"></a>|TimAllen set owner to @jesstess|
|-|-|

Thanks for doing the spell-checking and grammar editing, jesstess — it does read much more nicely now. The only edit I think is negative is line 83 where you avoided a trailing preposition: I think "the address the socket is 'connected' to" flows better than "the address to which the socket is 'connected'".

I'm prepared to call that a personal style issue, though. +1 for merging with or without that change.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_10"></a>|@jesstess set status to `closed`|
|-|-|

(In [28656]) Merge broken-udp-links-4274

Author: jesstess
Reviewers: thijs, TimAllen
Fixes: [#4274](https://github.com/twisted/twisted/issues/4274)

Fix broken API links, add markup, spellcheck, and rework some phrasing.
